### ISSUE TITLE
th175: stage name font

### ISSUE THREAD

**Issue by brliron** [thpatch/thcrap#182]
Replace the stage name font for better unicode support
![image](https://user-images.githubusercontent.com/10382983/149849543-415fc2ba-a356-4310-8a6a-8fb8a1a2f6c8.png)

**Comment by brliron** [comment]
Some points:
1. The stage number needs to be displayed with Touhou Biolinum, because it needs characters missing from the JP font.
2. The stage title (the thing right after the stage number) isn't translatable yet (#180), so it is still in Japanese, and can't be displayed with Touhou Biolimun. For now, it needs to be displayed with the JP font.
3. For aesthetic reasons, we may want to keep the same font for both texts.
4. For aesthetic reasons, English might want to keep the JP font.

With all that in mind, we can either change the font for the stage number, or both fonts. In both cases, it would go into th175/data/script/scene/subtitle.nut.jdiff.
```
For both fonts:
{
    "/Functions/2/Literals/26": "touhou_biolinum"
}
```

For the 1st font only:
```
{
    "/Functions/2": {
        "replace_instructions": {
            "78": "dload(s2, \"fontface\", s3, \"touhou_biolinum\")",
        },
        "add_literals": [
            "touhou_biolinum"
        ]
    }
}
```

**Comment by brliron** [comment]
Fixed in thpatch/thcrap-tsa@fd07fa25c0e846f2db58153cb662e7c71b88c14b
